hack/dockerized

```diff
@@ -12,9 +12,9 @@ fi
 
 fail_if_cri_bin_missing
 
-kubevirt_builder_version="2505072057-63d7190f20"
-default_kubevirt_builder_image="quay.io/kubevirt/builder:${kubevirt_builder_version}"
-kubevirt_cross_compile_image="quay.io/kubevirt/builder-cross:${kubevirt_builder_version}"
+kubevirt_builder_version="2505091401-8ab710cdb8"
+default_kubevirt_builder_image="quay.io/ffossemo/builder:${kubevirt_builder_version}"
+kubevirt_cross_compile_image="quay.io/ffossemo/builder-cross:${kubevirt_builder_version}"
```

Comment:
Needs to be reverted once image was pushed

Reply:
Perhaps we need another PR for it?

Reply:
YEs, I need to update with the one copied by @brianmcarey in the kv repo.
I was testing the CI

Reply:
Here we go :)